File: leetcode-problems/0058-length-of-last-word/solution.py
class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        s1=list(s.split(" "))
        for i in range((len(s1)-1),-1,-1):
            if len(s1[i])==0:
                s1.remove(s1[i])
        return len(s1[len(s1)-1])

File: leetcode-problems/0058-length-of-last-word/solution.cpp
class Solution {
public:
    int lengthOfLastWord(string s) {
        std::vector<std::string> l;
        std::string word;
        stringstream ss(s);
        while (ss>>word){
            l.push_back(word);
        }
        std::string le=l.back();
        return le.size();

    }
};
